tm_examples/detection_map.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+from torchmetrics import MAP
+
+preds = [
+    dict(
+        boxes=torch.Tensor([[258.0, 41.0, 606.0, 285.0]]),
```

Comment:
please finish the description what the four numbers are
```suggestion
        # The boxes keyword should contain an [N,4] tensor,
        # where N is the number of detected boxes and ...
        boxes=torch.Tensor([[258.0, 41.0, 606.0, 285.0]]),
```